sparkly/reader.py

```diff
@@ -214,6 +216,33 @@ def mysql(self, host, database, table, port=None, parallelism=None, options=None
 
         return self._basic_read(reader_options, options, parallelism)
 
+    def kafka(self, brokers, offset_ranges, key_deserializer, value_deserializer):
```

Comment:
is it possible to auto discover partitions and offsets? So we can have a signature like:
```
def kafka(self, host, topic, partitions=None, offsets=None, port=None,
    key_deserializer=_default_serializer, value_deserializer=_default_serializer)
```